getblocklab/block-lab
# Add component Jest tests

#### Changes
* This uses the [react-testing-library](https://testing-library.com/docs/react-testing-library/).
* These component tests don't look to be flaky. They've been passing when they should in this repo and in another [one of my plugins](https://github.com/kienstra/augmented-reality).
* Ideally, all of the components would have tests. But I think this PR is a small improvement.

#### Testing instructions
The only non-test files that this touches are `fetch-input.js` and `repeater-rows.js`. So this is just regression testing for those.

1. Create a Block Lab block with a Repeater
2. Populate the fields, and add a second row
3. Enter values for the fields
4. Ensure that the values save, and they appear in the `<ServerSideRender>` and the front-end
5. Repeat steps 1-4 for Post and Taxonomy fields
